extra_time_application/tests/test_extra_time.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (www.elico-corp.com)
+# Elico Proprietary License v1.0.
+# See LICENSE file for full copyright and licensing details.
+from odoo.tests import common
+
+
+class TestExtraTime(common.TransactionCase):
```

Comment:
you are not testing anything here, just covering the percentage.
For example, you need to test approval by user -> should fail